pkg/apis/kubeflow/v2beta1/default.go

```diff
@@ -53,6 +53,9 @@ func setDefaultsRunPolicy(policy *RunPolicy) {
 	if policy.CleanPodPolicy == nil {
 		policy.CleanPodPolicy = ptr.To(CleanPodPolicyNone)
 	}
+	if policy.ManagedBy == nil {
```

Comment:
Why are we defaulting the field? It is a decision in other Job CRDs not to default it, so I would suggest to keep it this way, for consistency. It can also be helpful for MultiKueue. If we default the field here, then the MultiKueue webhook will need to override the value, rather than just set.